### DISCUSSION TITLE
Repository not showing logs in dashboard since late July (renovate not operating either)

### DISCUSSION THREAD

**Discussion by bruxisma** [renovatebot/renovate#17354] [Request Help]
Hello,

I've run into an issue where a golang based repository is not showing logs since at least July 21st, when the last renovatebot PR was merged in. It is using the same configuration as several other golang repositories, all of which have their logs reported in the dashboard app.  The repository in question is [bruxisma/go-doze](https://github.com/bruxisma/go-doze). It is using the same configuration file as [bruxisma/go-improvmx](https://github.com/bruxisma/go-improvmx), both of which are simply extensions of [bruxisma/renovate](https://github.com/bruxisma/renovate). `go-doze` shows no logs as seen in the attached image below

![image](https://user-images.githubusercontent.com/63051/186053693-32c0f9e1-5767-4d9c-8040-72b0ee09444f.png)

Whereas `go-improvmx` shows a single run after having just been activated.

![image](https://user-images.githubusercontent.com/63051/186053793-e628cb2f-bae4-4e47-8839-665063b37fa6.png)

I had tried removing and re-adding the `go-doze` repository with no luck in changing the behavior. `go-doze` itself is a dependency of `go-improvmx`, and also `terraform-provider-improvmx` which has been using renovate for much longer 

![image](https://user-images.githubusercontent.com/63051/186054161-64bfbc54-ed13-4e4f-8581-2ce4a6526dc6.png)

Any help on this matter would be appreciated (I'm currently running into other issues with my golang configuration, but want to debug them separately)

**Comment by rarkins** [comment]
For some reason it was disabled due to a "not found" error and hadn't yet been retried. I enqueued it manually and it seems working now

**Reply by bruxisma** [reply]
Thank you! Can confirm it's showing logs on my end. Strange that it ended up in that state. 😅
